eyes-two/build.gradle

```diff
@@ -6,12 +6,12 @@ Properties properties = new Properties()
 properties.load(project.rootProject.file('local.properties').newDataInputStream())
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
+    compileSdkVersion 28
+    buildToolsVersion '28.0.3'
```

Comment:
I think this line and the other `buildToolsVersion` lines can be removed altogether with 28.